plugins/utils/comment-box.nix

```
@@ -0,0 +1,150 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib) types;
```

```suggestion
  inherit (lib) mkAttrsOf mkBool mkInt mkStr types;
```
Might need to be something like this for all the others that were from the `with lib`

---

Sorry, I didn't see you used the inherit from defaultNullOpts in a with block.. you can revert that last one.

---

No worries, both are fine to me. I'll keep it that way since I find it to be easier for the eyes too